src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/UpdateContactDetailsService.java

```diff
@@ -97,11 +116,38 @@ void removeApplicantSolicitorDetails(Map<String, Object> caseData, String caseTy
         }
     }
 
+    void removeApplicantSolicitorDetails(FinremCaseData caseData, CaseType caseType) {
+        boolean isContested = CaseType.CONTESTED.equals(caseType);
+        ContactDetailsWrapper contactDetailsWrapper = caseData.getContactDetailsWrapper();
+
+        if (contactDetailsWrapper.getApplicantRepresented().isNoOrNull()) {
```

Comment:
should check null on `getApplicantRepresented`?

Reply:
isNoOrNull should handle the null check for us, and prior to this call I have added in and NPE safeguards in the handler for ContactDetailsWrapper.

Reply:
_isNoOrNull should handle the null check for us_

So if contactDetailsWrapper.getApplicantRepresented == null then what happens when your code is executed?

Reply:
Those "null" related methods in YesOrNo seem very suspect and misleading.

Reply:
I see the error of my ways

Reply:
I think you are meant to call them like
`YesOrNo.isNoOrNull(contactDetailsWrapper.getApplicantRepresented())`